airflow/airflow.go

```diff
@@ -118,14 +119,21 @@ func Init(path, airflowImageName, airflowImageTag string) error {
 		".astro/dag_integrity_exceptions.txt":  "# Add dag files to exempt from parse test below. ex: dags/<test-file>",
 	}
 
-	// Initailize directories
-	if err := initDirs(path, dirs); err != nil {
-		return errors.Wrap(err, "failed to create project directories")
-	}
+	if template != "" {
```

Comment:
This placement is mixed in with the existing non-template init flow, which is a bit confusing. Can we add the `template != ""` check at the top of the function and have it return early so we don't do unnecessary things like construct the `files` map?